saltproc/depcode.py

```diff
@@ -175,6 +165,29 @@ def __init__(self,
                  active_cycles=20,
                  inactive_cycles=20):
         """Initializes the DepcodeSerpent object.
+
+           Parameters
+           ----------
+           exec_path : str
+               Path to Serpent2 executable.
+           template_path : str
+               Path to user input file for Serpent2.
+           input_path : str
+               Name of input file for Serpent2 rerunning.
+           iter_matfile : str
+               Name of iterative, rewritable material file for Serpent2
+               rerunning. This file is modified during  the simulation.
```

Comment:
Consider adding the optional tag to these parameters

Reply:
I was thinking the default values would typically work, but yeah I think you're right that these aren't really optional.